YOLayout/UIKit/YOView.h

```diff
@@ -29,6 +29,9 @@
  */
 @interface YOView : UIView <YOLayoutView> { }
 
+@property YOLayout *viewLayout;
+
+// Deprecated
 @property YOLayout *layout;
```

Comment:
Add `DEPRECATED_ATTRIBUTE`

Or honestly, probably OK to just remove. We haven't struck it big quite yet :)